### PR TITLE
Copy edit preface section

### PR DESCRIPTION
**Pull request by catherinetcai**
@tuhdo I did a quick copy edit of the preface section. I just wanted to get your take on how it was stylistically etc. If you do like it, let me know. I'm happy to keep helping you copy edit this book :)

### CODE REVIEW

**Comment by tuhdo** [comment]
@catherinetcai Could you give more details what changes did you make? I cannot see the diff, and when I tried to build it, the layout seems the same.

**Comment by catherinetcai** [comment]
@tuhdo I pushed again to my fork. It's just the first two pages of the book of the preface. Pull again to see the changes.

**Comment by tuhdo** [comment]
The changes are nice. Could you rebase against the current master? Or if you don't mind, I can manually fix it myself. Thanks.

**Comment by catherinetcai** [comment]
Happy to do so. Would you like me to continue as you write?

**Comment by tuhdo** [comment]
Sure keep going. I will wait for your PR to be merged before others (if any appears).

**Comment by catherinetcai** [comment]
@tuhdo Updated. I'll keep moving forward :)

File: book_src/Operating System From 0 to 1.lyx [hunk 1]
```diff
@@ -2643,6 +2644,10 @@ status collapsed
 \end_inset
 
 
+\begin_inset Separator latexpar
```
Comment (tuhdo) [inline book_src/Operating System From 0 to 1.lyx]: It seems there are some incompatibilities between Lyx 2.1 and 2.2.  Could you stage only the edited text and not the whole file?

File: book_src/Operating System From 0 to 1.lyx [hunk 2]
```diff
@@ -436,70 +432,72 @@ If that is the case, this book is for you.
  scratch! Yes, from scratch, without going through any existing operating
  system layer to comfort yourself that you are an operating system developer.
  You may ask, 
+
 \begin_inset Quotes eld
 \end_inset
 
-Isn't it more practical to learn the internals of Linux?
+Isn't it more practical to learn the internals of Linxus?
```
Comment (tuhdo) [inline book_src/Operating System From 0 to 1.lyx]: Typo here.

File: book_src/Operating System From 0 to 1.lyx (hunk 1 above)
Comment (catherinetcai) [inline book_src/Operating System From 0 to 1.lyx]: Hmm, yeah. I'll try and copy the edited text and paste it in Vim. I'll also see if I can just download 2.1.

Comment (tuhdo) [inline book_src/Operating System From 0 to 1.lyx]: I tried removing them with Emacs, but Lyx still gives a warning about unknown insets. I think it's best to use Lyx 2.1 at the moment. Aside from the warning, the generated PDF looks normal. But I will update to let people use Lyx 2.1.

**Comment by catherinetcai** [comment]
@tuhdo - Ok, moved down to version 2.1.3. I'm apparently missing some LaTex modules, but hopefully that shouldn't impact the editing process too much?

**Comment by tuhdo** [comment]
Thank you very much. At the moment, it seems it is better if you use Vim and edit the content, so the layout won't be messed up. Even with Lyx 2.1, it still changes the Latex markups of the book. Sorry for the inconvenience.

**Pull request merged**